xym/xym.py

```diff
@@ -143,12 +143,12 @@ def get_extracted_models(self, force_revision_pyang, force_revision_regexp):
             models.extend(self.extracted_models)
             for model in models:
                 if force_revision_pyang:
-                    command = '/usr/local/bin/pyang -f name-revision "' + self.dst_dir + '/' + model + '"'
+                    command = '/usr/local/bin/pyang -f name --name-print-revision "' + self.dst_dir + '/' + model + '"'
```

Comment:
Does `-f name --name-print-revision` depend on a specific minimum version of `pyang`?